apps/server/src/shared/testing/user-role-permissions.ts

```diff
@@ -5,297 +5,290 @@
 import { Permission } from '@shared/domain';
 
 export const userPermissions = [
-	'BASE_VIEW',
-	'DASHBOARD_VIEW',
-	'TOOL_VIEW',
-	'HOMEWORK_CREATE',
-	'PASSWORD_EDIT',
-	'FOLDER_DELETE',
-	'FOLDER_CREATE',
-	'FILE_DELETE',
-	'FILE_CREATE',
-	'FILE_MOVE',
-	'ACCOUNT_EDIT',
-	'CALENDAR_VIEW',
-	'CALENDAR_EDIT',
-	'CALENDAR_CREATE',
-	'FEDERALSTATE_VIEW',
-	'HELPDESK_CREATE',
-	'TOPIC_VIEW',
-	'LINK_CREATE',
-	'NEWS_VIEW',
-	'NOTIFICATION_VIEW',
-	'NOTIFICATION_EDIT',
-	'NOTIFICATION_CREATE',
-	'PWRECOVERY_VIEW',
-	'PWRECOVERY_EDIT',
-	'PWRECOVERY_CREATE',
-	'RELEASES_VIEW',
-	'ROLE_VIEW',
-	'COURSEGROUP_CREATE',
-	'COURSEGROUP_EDIT',
-	'SYSTEM_VIEW',
-	'HOMEWORK_VIEW',
-	'HOMEWORK_EDIT',
-	'COMMENTS_VIEW',
-	'COMMENTS_CREATE',
-	'COMMENTS_EDIT',
-	'SUBMISSIONS_VIEW',
-	'SUBMISSIONS_CREATE',
-	'SUBMISSIONS_EDIT',
-	'FILESTORAGE_VIEW',
-	'FILESTORAGE_EDIT',
-	'FILESTORAGE_CREATE',
-	'CONTENT_VIEW',
-	'CONTENT_NON_OER_VIEW',
-	'FILESTORAGE_REMOVE',
-	'TEAM_VIEW',
-	'PRIVACY_VIEW',
-	'CLASS_VIEW',
-	'COURSE_VIEW',
-	'LERNSTORE_VIEW',
+	Permission.BASE_VIEW,
```

Comment:
@CeEv Why do we need this file? Or why is it in the shared/testing folder?

Reply:
It is used for the generated user over the UserAndAccountTestFactory, that it is used for the api tests.